pkgs/tools/admin/kics/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, buildGoModule, fetchFromGitHub, lib, installShellFiles }:
+
+buildGoModule rec {
+  pname = "kics";
+  version = "1.5.12";
+
+  src = fetchFromGitHub {
+    owner = "Checkmarx";
+    repo = "kics";
+    rev = "v${version}";
+    sha256 = "sha256-jHspStyjq5T9jzYDRYaf2gOI4F/X+h4nDn0PFUOHoBY=";
+  };
+
+  vendorSha256 = "sha256-q5NuuP04kOoLVj210s17fIW2cxrsC/tAyET8YYGai0M=";
+
+  doCheck = false;
```

Comment:
Please provide a comment on why is this disabled.

Reply:
removed, it was copied from different pkg.